app/helpers/scripts/versions_helper.rb

```diff
@@ -2,24 +2,25 @@ module Scripts::VersionsHelper
   def version_user_for version
     originator = version.paper_trail_originator || version.whodunnit
 
-    User.find originator if originator
+    @users ||= {}
+    @users[originator] ||= User.find originator if originator
   end
 
   def version_change_for version
-    version.event == 'create' ? t('scripts.versions.index.event.create') : (version.reify&.change || '-')
+    change = version.object_changes['change']&.last || (version.reify&.change || '-')
+
+    change += " (#{t('scripts.versions.index.event.create')})" if version.event == 'create'
```

Comment:
Queda medio largo usado como modificador, lo pondría así:

```suggestion
    if version.event == 'create'
      change += " (#{t 'scripts.versions.index.event.create'})"
    end
```

(Nótese el Seattle dentro del interpolado).